### PR TITLE
Update docs with project attributes permissions description

### PR DESCRIPTION
**Pull request by dombesz**
Follow up PR for https://community.openproject.org/wp/50844 to describe the newly added permissions in the user and system administrator guides.

### CODE REVIEW

**Comment by dombesz** [comment]
Hi @corinnaguenther  and @MayaBerd ,

I added some description for the newly added project attributes. Please feel free to add more if necessary, review or merge. Thanks!

File: docs/user-guide/project-overview/README.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ Open the project overview by navigating to **Overview** in the project menu on t
 
 You will see a list of all available project attributes in a pane on the right side of of your Project overview page. They may be grouped in sections.
 
+> [!TIP]
+>
```
Comment (MayaBerd) [inline docs/user-guide/project-overview/README.md]:
```suggestion
```

File: docs/user-guide/project-overview/README.md [hunk 2]
```diff
@@ -38,6 +38,12 @@ Open the project overview by navigating to **Overview** in the project menu on t
...
+> Your view of the project attributes may vary depending on on your  [roles and permissions in OpenProject](../../system-admin-guide/users-permissions/roles-permissions/). 
+>
```
Comment (MayaBerd) [inline docs/user-guide/project-overview/README.md]:
```suggestion
```

File: docs/user-guide/project-overview/README.md [hunk 3]
```diff
@@ -46,15 +52,18 @@ To edit the value of any visible project attribute, click on the **Edit** (penci
...
 
 ### Project attribute settings 
 
 To adjust the the project attribute settings for a specific project click the **More** (three dots) icon and select *Manage project attributes*. This will lead you directly to the [project attribute settings](../projects/project-settings/project-attributes/).
 
 ![Link to project attribute settings from project overview page in OpenProject](openproject_user_guide_project_overview_project_attributes_settings.png)
 
->Note: This option is always available to instance and project administrators. It can also be activated for specific roles by enabling the *select_project_attributes* permission for that role via the [Roles and permissions page](../../system-admin-guide/users-permissions/roles-permissions/) in the administrator settings.
+>[!NOTE]
+>
```
Comment (MayaBerd) [inline docs/user-guide/project-overview/README.md]:
```suggestion
```

File: docs/user-guide/project-overview/README.md [hunk 4]
```diff
@@ -69,7 +78,9 @@ You can archive a project directly from the project overview page. To do that cl
 
 ![Archive a project on the project overview page in OpenProject](openproject_user_guide_project_overview_archive_project.png)
 
->Note: This option is always available to instance and project administrators. It can also be activated for specific roles by enabling the *archive_project* permission for that role via the [Roles and permissions page](../../system-admin-guide/users-permissions/roles-permissions/) in the administrator settings.
+>[!NOTE]
+>
```
Comment (MayaBerd) [inline docs/user-guide/project-overview/README.md]:
```suggestion
```

File: docs/user-guide/project-overview/README.md [hunk 5]
```diff
@@ -138,7 +149,9 @@ The description can be added or changed in the [project settings](../projects/pr
 
 ### Project details widget
 
-> **Important note**: Project details widget was replaced by [Project attributes](#project-attributes) in [OpenProject 14.0](../../release-notes/14-0-0/) and will be removed in the future.
+> [!IMPORTANT]
+>
```
Comment (MayaBerd) [inline docs/user-guide/project-overview/README.md]:
```suggestion
```

File: docs/user-guide/projects/project-settings/project-attributes/README.md [hunk 6]
```diff
@@ -8,11 +8,17 @@ keywords: project attributes, project settings, enable, disable, project admin
...
 # Project attributes
 
-**Project attributes**are a set of project-level custom fields that let you display certain types of information relevant to your project in the [Project overview](../../../project-overview) page.
+**Project attributes** are a set of project-level custom fields that let you display certain types of information relevant to your project in the [Project overview](../../../project-overview) page.
 
 This guide is aimed at project administrators who want to enable or disable certain project attributes for their project.
 
->**Note:**  If you are an instance admin and would like to create, modify or add project attributes, please read our [admin guide to project attributes](../../../../system-admin-guide/projects/project-attributes).
+> [!NOTE]
+>
```
Comment (MayaBerd) [inline docs/user-guide/projects/project-settings/project-attributes/README.md]:
```suggestion
```

File: docs/user-guide/projects/project-settings/project-attributes/README.md [hunk 7]
```diff
@@ -8,11 +8,17 @@ keywords: project attributes, project settings, enable, disable, project admin
...
+> Alongside the project administrators, other users can also be given the permission to enable/disable project attributes by enabling the **Select project attributes** permission for their roles.
+
+>[!TIP]
+>
```
Comment (MayaBerd) [inline docs/user-guide/projects/project-settings/project-attributes/README.md]:
```suggestion
```

File: docs/user-guide/projects/project-settings/project-attributes/README.md [hunk 8]
```diff
@@ -26,4 +32,6 @@ You can also use the **Enable all** and **Disable all** buttons visible to the r
 
 If your instance has a particularly long list of project attributes, you can use the search bar at the top to find specific ones.
 
->**Note**: The project settings page for project attributes only lets you enable or disable certain attributes. It does *not* let you set the values for each. To do this, go to the [Project Overview](../../../project-overview) page.
+>[!TIP]
+>
```
Comment (MayaBerd) [inline docs/user-guide/projects/project-settings/project-attributes/README.md]:
```suggestion
```

**Pull request merged**